confgenerator/confgenerator.go

```diff
@@ -52,7 +52,7 @@ func googleCloudExporter(userAgent string) otel.Component {
 				// descriptors to be created implicitly with new time series.
 				"skip_create_descriptor": true,
 				// Omit instrumentation labels, which break agent metrics.
-				"instrumentation_library_labels": false,
+				"instrumentation_library_labels": instrumentationLabels,
 				// Omit service labels, which break agent metrics.
 				"service_resource_labels": false,
```

Comment:
From the offline discussions it sounded like the service_resource_labels were also supposed to be turned on for the `OTEL` exporter type?

https://github.com/GoogleCloudPlatform/opentelemetry-operations-collector/pull/150#issuecomment-1416170876 for some context

Is that happening at another time or not needed at all for 3P metrics?

Reply:
It needs to be turned on separately because we need to do more work on making the data correct first (and it's not on the critical path for `instrumentation_scope`).